component/stream/read_write.go

```diff
@@ -173,9 +173,8 @@ func (rw *ReadWriteCache) TruncateFile(options internal.TruncateFileOptions) err
 	err := rw.NextComponent().TruncateFile(options)
```

Comment:
In TruncateFile we are changing the file on container, shouldn't we invalidate our local caches here ?

Reply:
I thought we had agreed no modifying other handles on read_write?